build/.vsts-ci.yml

```diff
@@ -300,7 +300,7 @@ jobs:
     condition: always()
 - job: wolfssl
   variables:
-    _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/c-linux-wolfssl"
+    _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/wolfssl-1804"
```

Comment:
Minor nag on this lol. "1804" for anyone not acquainted is a little cryptic.
Is it related to ubuntu 1804? If yes, what does it mean being in a pool named "Hosted Ubuntu 1604"?
c-linux-wolfssl seems preferable. Again, super minor thing.